orchagent/aclorch.cpp

```diff
@@ -2913,6 +2920,12 @@ sai_object_id_t AclOrch::getTableById(string table_id)
 {
     SWSS_LOG_ENTER();
 
+    if (table_id.empty())
+    {
+        SWSS_LOG_WARN("table_id is empty");
+        return SAI_NULL_OBJECT_ID;
+    }
+
```

Comment:
> I think if you have checked the table name in the `doAclRuleTask` function, you don't need to check it again here in this function.

This function was called by different places, to avoid any endless recursive call, I prefer we check here too.